src/Mock/MockedDriver.php

```diff
@@ -20,7 +20,7 @@
 use Exception;
 
 /**
- * Class MockedDriver
+ * Class MockedDriver.
```

Comment:
Comments like these are totally useless. I highly recommend removing them everywhere.

Reply:
Thanks for your recommendation. I like these redundant comments :+1: